README.md

```diff
@@ -30,6 +30,7 @@ Python SDK为[FISCO BCOS](https://github.com/FISCO-BCOS/FISCO-BCOS/tree/master)
 ### 依赖软件
 
 - **Ubuntu**: `sudo apt install -y zlib1g-dev libffi6 libffi-dev wget git`
+- **Alpine**: `apk add --no-cache gcc g++ python python-dev py-pip mysql-dev linux-headers libffi-dev openssl-dev git wget bash`
```

Comment:
Why should install `mysql-dev` and `linux-headers`?

Reply:
> Why should install `mysql-dev` and `linux-headers`?

That's the production environment for python.I'm loading this as a python workspace because I like to build projects in a docker container.If the removal of these two has no effect on the operation of the web framework (flask), then I will consider canceling it. 
emmm Diy environment. hahahaha😂

Reply:
> Why should install `mysql-dev` and `linux-headers`?

It's worth noting that alpine doesn't have a bash with its own sh, so try to switch to the bash interface when you're working.
Ps: it may be a little more complicated, but it can be exchanged for a lighter box